diff -pruN libdrm-2.4.49.orig/include/drm/qxl_drm.h libdrm-2.4.49/include/drm/qxl_drm.h
--- libdrm-2.4.49.orig/include/drm/qxl_drm.h	2013-11-23 06:51:11.230361834 +0000
+++ libdrm-2.4.49/include/drm/qxl_drm.h	2013-11-23 06:52:16.104005907 +0000
@@ -25,7 +25,7 @@
 #define QXL_DRM_H
 
 #include <stddef.h>
-#include "drm/drm.h"
+#include "drm.h"
 
 /* Please note that modifications to all structs defined here are
  * subject to backwards-compatibility constraints.
