diff -Nru dovecot-2.1.6.orig/doc/example-config/conf.d/10-master.conf dovecot-2.1.6/doc/example-config/conf.d/10-master.conf
--- dovecot-2.1.6.orig/doc/example-config/conf.d/10-master.conf	2012-05-07 08:36:14.788627433 +0200
+++ dovecot-2.1.6/doc/example-config/conf.d/10-master.conf	2012-05-07 08:36:28.178142765 +0200
@@ -117,3 +117,10 @@
     #group = 
   }
 }
+
+service tcpwrap {
+   unix_listener login/tcpwrap {
+     mode = 0666
+   }
+}
+
diff -Nru dovecot-2.1.6.orig/doc/example-config/conf.d/10-ssl.conf dovecot-2.1.6/doc/example-config/conf.d/10-ssl.conf
--- dovecot-2.1.6.orig/doc/example-config/conf.d/10-ssl.conf	2012-05-07 08:36:14.788627433 +0200
+++ dovecot-2.1.6/doc/example-config/conf.d/10-ssl.conf	2012-05-07 08:36:28.178142765 +0200
@@ -9,8 +9,8 @@
 # dropping root privileges, so keep the key file unreadable by anyone but
 # root. Included doc/mkcert.sh can be used to easily generate self-signed
 # certificate, just make sure to update the domains in dovecot-openssl.cnf
-ssl_cert = </etc/ssl/certs/dovecot.pem
-ssl_key = </etc/ssl/private/dovecot.pem
+ssl_cert = </etc/ssl/certs/dovecot.crt
+ssl_key = </etc/ssl/keys/dovecot.key
 
 # If key file is password protected, give the password here. Alternatively
 # give it when starting dovecot with -p parameter. Since this file is often
diff -Nru dovecot-2.1.6.orig/doc/example-config/conf.d/auth-system.conf.ext dovecot-2.1.6/doc/example-config/conf.d/auth-system.conf.ext
--- dovecot-2.1.6.orig/doc/example-config/conf.d/auth-system.conf.ext	2012-05-07 08:36:14.788627433 +0200
+++ dovecot-2.1.6/doc/example-config/conf.d/auth-system.conf.ext	2012-05-07 08:36:28.178142765 +0200
@@ -7,12 +7,12 @@
 # PAM is typically used with either userdb passwd or userdb static.
 # REMEMBER: You'll need /etc/pam.d/dovecot file created for PAM
 # authentication to actually work. <doc/wiki/PasswordDatabase.PAM.txt>
-passdb {
-  driver = pam
+#passdb {
+  #driver = pam
   # [session=yes] [setcred=yes] [failure_show_msg=yes] [max_requests=<n>]
   # [cache_key=<key>] [<service name>]
   #args = dovecot
-}
+#}
 
 # System users (NSS, /etc/passwd, or similiar).
 # In many systems nowadays this uses Name Service Switch, which is
@@ -26,11 +26,11 @@
 # Shadow passwords for system users (NSS, /etc/shadow or similiar).
 # Deprecated by PAM nowadays.
 # <doc/wiki/PasswordDatabase.Shadow.txt>
-#passdb {
-  #driver = shadow
+passdb {
+  driver = shadow
   # [blocking=no]
   #args = 
-#}
+}
 
 # PAM-like authentication for OpenBSD.
 # <doc/wiki/PasswordDatabase.BSDAuth.txt>
diff -Nru dovecot-2.1.6.orig/doc/example-config/dovecot.conf dovecot-2.1.6/doc/example-config/dovecot.conf
--- dovecot-2.1.6.orig/doc/example-config/dovecot.conf	2012-05-07 08:36:14.785294220 +0200
+++ dovecot-2.1.6/doc/example-config/dovecot.conf	2012-05-07 08:36:28.178142765 +0200
@@ -24,6 +24,7 @@
 # If you want to specify non-default ports or anything more complex,
 # edit conf.d/master.conf.
 #listen = *, ::
+listen = *
 
 # Base directory where to store runtime data.
 #base_dir = /var/run/dovecot/
@@ -44,7 +45,7 @@
 #login_trusted_networks =
 
 # Sepace separated list of login access check sockets (e.g. tcpwrap)
-#login_access_sockets = 
+login_access_sockets = tcpwrap
 
 # With proxy_maybe=yes if proxy destination matches any of these IPs, don't do
 # proxying. This isn't necessary normally, but may be useful if the destination
