diff -Nru libxml2-2.9.0.orig/python/Makefile.in libxml2-2.9.0/python/Makefile.in
--- libxml2-2.9.0.orig/python/Makefile.in	2012-09-14 10:22:28.919971395 +0200
+++ libxml2-2.9.0/python/Makefile.in	2012-09-14 10:23:40.717353028 +0200
@@ -82,7 +82,7 @@
 LTLIBRARIES = $(python_LTLIBRARIES)
 am__DEPENDENCIES_1 =
 @WITH_PYTHON_TRUE@libxml2mod_la_DEPENDENCIES =  \
-@WITH_PYTHON_TRUE@	$(top_builddir)/libxml2.la \
+@WITH_PYTHON_TRUE@	$(libdir)/libxml2.la \
 @WITH_PYTHON_TRUE@	$(am__DEPENDENCIES_1) $(am__DEPENDENCIES_1) \
 @WITH_PYTHON_TRUE@	$(am__DEPENDENCIES_1)
 am__libxml2mod_la_SOURCES_DIST = libxml.c libxml_wrap.h libxml2-py.h \
@@ -416,7 +416,7 @@
 
 @WITH_PYTHON_TRUE@python_LTLIBRARIES = libxml2mod.la
 @WITH_PYTHON_TRUE@libxml2mod_la_SOURCES = libxml.c libxml_wrap.h libxml2-py.h libxml2-py.c types.c
-@WITH_PYTHON_TRUE@libxml2mod_la_LIBADD = $(top_builddir)/libxml2.la $(CYGWIN_EXTRA_PYTHON_LIBADD) $(WIN32_EXTRA_PYTHON_LIBADD) $(PYTHON_LIBS) -lpython$(PYTHON_VERSION)
+@WITH_PYTHON_TRUE@libxml2mod_la_LIBADD = $(libdir)/libxml2.la $(CYGWIN_EXTRA_PYTHON_LIBADD) $(WIN32_EXTRA_PYTHON_LIBADD) $(PYTHON_LIBS) -lpython$(PYTHON_VERSION)
 @WITH_PYTHON_TRUE@libxml2mod_la_LDFLAGS = $(CYGWIN_EXTRA_LDFLAGS) $(WIN32_EXTRA_LDFLAGS) -module -avoid-version
 @WITH_PYTHON_TRUE@BUILT_SOURCES = libxml2-export.c libxml2-py.h libxml2-py.c
 @WITH_PYTHON_TRUE@dist_python_DATA = \
